common/archive-impl/impl2/src/webapp/WEB-INF/components.xml

```diff
@@ -33,6 +33,7 @@
 				<value>WebService</value>
 				<value>LessonBuilderEntityProducer</value>
 				<value>PollListManager</value>
+				<value>rubrics</value>
```

Comment:
shouldn't this match what is in default.sakai.properties?

Reply:
It matches what getLabel returns in RubricsServiceImpl

Reply:
RubricsService?

Reply:
I think i prefer just "rubrics". I don't really see the point in using an internal name of a class. There are already several styles of naming in there so I think "rubrics" is clearer. Also, I don't know how to change it from the producer's label :)